packages/manager-components/src/components/notifications/notifications.component.tsx

```diff
@@ -20,14 +19,17 @@ interface NotificationProps {
 export const Notifications: FC<NotificationProps> = ({
   clearAfterRead = true,
 }) => {
-  const location = useLocation();
-  const [originLocation] = useState(location);
-  const { notifications, clearNotifications } = useNotifications();
+  const { notifications, clearNotification } = useNotifications();
 
   useEffect(() => {
-    if (clearAfterRead && originLocation.pathname !== location.pathname)
-      clearNotifications();
-  }, [clearAfterRead, location.pathname]);
+    const timer = setTimeout(() => {
```

Comment:
i think this will create a lot of unecessary timeout functions each time notifications array changes (when a new notification is pushed for example)

you probably don't want to create multiple timeouts (even if only the first one will do something), it could be avoided by adding an attribute in notifcation type to check if the clear is planned for example